sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -86,22 +86,26 @@ int pollSocketUntilEventOrTimeout(
 }
 
 #ifdef WINDOWS
-// Windows needs this after every write to socket or peformance would be reduced to 1/4 for
+// Windows needs this after every write to socket or performance would be reduced to 1/4 for
```

Comment:
Since you are fixing these, can you fix the across the whole file?

conentLength 
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L501
connet_only
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L997
containst 
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L855
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L930
delimeter 
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L903
everyting
https://github.com/Azure/azure-sdk-for-cpp/blob/9120f797d582f5262269793222c477f7608654eb/sdk/core/azure-core/src/http/curl/curl.cpp#L622

... and any others you might find (specific to this file)

Use extensions like these to help you spellcheck in VS Code/browser, if they help (fair warning, they will have lots of false positives so you need to create a dictionary of programming words to ignore):
https://www.site-spell.com/
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker